uhyve-interface/src/v1/parameters.rs

```diff
@@ -57,14 +59,6 @@ pub struct CmdvalParams {
 	pub envp: GuestPhysAddr,
 }
 
-/// Parameters for a [`Exit`](crate::v1::Hypercall::Exit) hypercall.
-#[repr(C, packed)]
-#[derive(Debug, Copy, Clone)]
-pub struct ExitParams {
-	/// The return code of the guest.
-	pub arg: i32,
-}
-
```

Comment:
@fogti, you moved a bunch of parameters to `v1`, which is perhaps more elegant in a way as it reduces the amount of things we have to explicitly change (by prefixing `v1::`), but does this solve e.g. how the kernel (currently on `v1`) imports `WriteParams` and `ReadParams` directly (i.e. technical debt for the future, should we decide to e.g. add more to `ExitParams`)?

I thought that "using `v1` by default" through re-exports has its benefits, but it might introduce "accidents". After all, the references will break once we bump the `uhyve-interface` version.

P.S. `ExitParams` is not directly used in `v2` - converting to a `i32` introduces a bug though where the sign is removed (hence why the `panic` integration test was failing).